accord-core/src/test/java/accord/utils/MessageTask.java

```diff
@@ -165,7 +165,7 @@ private static MessageTask of(Node originator, Collection<Node.Id> recipients, A
     public static MessageTask begin(Node originator, Collection<Node.Id> recipients, AgentExecutor executor, String desc, NodeProcess process)
     {
         MessageTask task = of(originator, recipients, executor, desc, process);
-        task.run();
+        executor.execute(task);
```

Comment:
begin can be called in a different store, so need to run the task in the correct store